### PR TITLE
Fix typo in CCC use case section

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fix a typo in the description: it's not the consumer of the attestation results but the consumer of the results of the confidential computation that we are concerned with here.

### CODE REVIEW

File: draft-fossati-tls-attestation.md [hunk 1]
```diff
@@ -815,7 +815,7 @@ including anti-money laundering, drug development in healthcare, contact
 tracing in pandemic times, etc.
 
 In such scenarios, the users (e.g., the party providing the data input for the
-computation, the consumer of the computed attestation results, the party
+computation, the party consuming the computed results, or the party
```
Comment (yogeshbdeshpande) [inline draft-fossati-tls-attestation.md:818]: I think, we need to work more on the example here! Suddenly we introduced the term `party providing a proprietary ML model` without setting the context.

Comment (muhammad-usama-sardar) [reply draft-fossati-tls-attestation.md:818]: Agree with @yogeshbdeshpande. There is no mention of ML model before this point in time. A little more context should be provided here. 
Additionally, the abbreviation ML is not spelled out.

Comment (thomas-fossati) [reply draft-fossati-tls-attestation.md:818]: Sure, the examples can always be improved.  The scope of this PR is narrower though: it's only meant to fix a bug in the actors' description.

Comment (thomas-fossati) [reply draft-fossati-tls-attestation.md:818]: > Additionally, the abbreviation ML is not spelled out.

fixed in 459df76cf06ea8a17d140300ccf405d33df9dd90

Comment (thomas-fossati) [reply draft-fossati-tls-attestation.md:818]: > I think, we need to work more on the example here! Suddenly we introduced the term `party providing a proprietary ML model` without setting the context.

fixed in d82eceed8438f3b44cde0170330a049134964243

**Review by muhammad-usama-sardar — APPROVED** [review]

File: draft-fossati-tls-attestation.md [hunk 2]
```diff
@@ -812,10 +812,10 @@ In this example, a confidential workload is executed on computational
 resources hosted at a cloud service provider.  This is a typical
 scenario for secure, privacy-preserving multiparty computation,
 including anti-money laundering, drug development in healthcare, contact
```
Comment (yogeshbdeshpande) [inline draft-fossati-tls-attestation.md:814]:
```suggestion
resources hosted at a cloud service provider.  This is a typical
scenario that requires secure, privacy-preserving multiparty computation,
environment for the workload. Typical workload examples include anti-money laundering,
drug development in healthcare, contact tracing in pandemic times,
Machine Learning (ML) model training, etc.
```

**Review by yogeshbdeshpande — COMMENTED** [review]
see a slightly more clear text on the left pane

File: draft-fossati-tls-attestation.md (hunk 2 above)
Comment (thomas-fossati) [reply draft-fossati-tls-attestation.md:814]: I am not sure the suggested text is correct.  In particular: _"This is a typical
scenario that requires secure, privacy-preserving multiparty computation,
environment for the workload."_ seems backwards.

**Pull request merged**